### PR TITLE
Retrofuture Space Plane Parts

### PR DESCRIPTION
**Pull request by linuxgurugamer**
Closes #4793

### CODE REVIEW

**Comment by linuxgurugamer** [comment]
Please check the filter line

File: NetKAN/retrofuturespaceplaneparts.netkan [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+      "name": "ModuleManager",
+      "name": "FirespitterCore"
+    }
+  ],
+  "install": [
+    {
+      "find": "RetroFuture",
+      "install_to": "GameData",
+      "filter"     : "Firespitter"
```
Comment (politas) [inline NetKAN/retrofuturespaceplaneparts.netkan]: "filter":      "Firespitter.dll"

**Comment by linuxgurugamer** [comment]
!rebuild

**Comment by linuxgurugamer** [comment]
!restart

**Comment by techman83** [comment]
#rebuild

**Pull request merged**